### ISSUE TITLE
Open a Firefox window while Chrome is my prefered browser

### ISSUE THREAD

**Issue by Nolaan** [birdieapp/birdie#112]
Hello,
I'm running KDE on Fedora 21 up to date. I've set chrome to be my default browser. However Birdie discard my preference and opens Firefox, and there's no way to customize it.

**Comment by jschnurr** [comment]
Birdie inherits this behavior from the use of [PyGTK](https://en.wikipedia.org/wiki/PyGTK).

From what I can tell, GTK+ processes URL clicks via the [gtk.show_uri](http://www.pygtk.org/pygtk2reference/gtk-functions.html#function-gtk--show-uri) function, which resolves the 'default' browser using the x-scheme-handler/http attribute in ~/.local/share/applications/mimeapps.list.

This is different than the way other applications identify the default browser on my Mint KDE system.

For example, my default browser is Google Chrome:
```bash
$ xdg-settings get default-web-browser        
google-chrome.desktop
```

However, my x-scheme-handler/http settings are set to Firefox:
```bash
$ xdg-mime query default x-scheme-handler/http
firefox.desktop
$ xdg-mime query default x-scheme-handler/https
firefox.desktop
```

To resolve it, set the x-scheme-handler for http and https to your preferred browser, as follows:
```bash
$ xdg-mime default google-chrome.desktop x-scheme-handler/http
$ xdg-mime default google-chrome.desktop x-scheme-handler/https
```

It may also be possible for GTK+ to be set to use default-web-browser on KDE.  If so, please reply or submit a PR.  That would fix it permanently.

**Comment by nathandyer** [comment]
Hi! This behavior is fixed in Birdie 2.0, which is written in Vala (and therefore doesn't suffer from the same inherited PyGTK behavior issues). Thank you for the report!
